library/core/src/lib.rs

```diff
@@ -82,6 +82,7 @@
 #![feature(const_fn_union)]
 #![feature(const_generics)]
 #![feature(const_option)]
+#![feature(const_precise_live_drops)]
```

Comment:
What is this? I'm in the const-eval GH group and I don't think I ever saw this or https://github.com/rust-lang/rust/issues/73255 ever before.^^